### PR TITLE
Fixed android app crash with API level 29

### PR DESCRIPTION
**Pull request by 0x1bitcrack3r**
### Contributing checklist
- [ ] Code must follow existing styling conventions
- [ ] Added a descriptive commit message

### Solves issue(s)

https://github.com/opentok/opentok-react-native/issues/455

### CODE REVIEW

File: android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java [hunk 1]
```diff
@@ -26,8 +26,9 @@ protected OTSubscriberLayout createViewInstance(ThemedReactContext reactContext)
 
     @ReactProp(name = "streamId")
     public void setStreamId(OTSubscriberLayout view, String streamId) {
-
+        if(streamId!=null){
```
Comment (igorwojda) [inline android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java:29]: To keep formatting consistent we should add spaces around `!=`, so:

`if(streamId != null) {`

**Review by igorwojda — CHANGES_REQUESTED** [review]
One small formatting change.
In he future more info in PR desc would be help full eg. crash sack trace

File: android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java (hunk 1 above)
Comment (0x1bitcrack3r) [reply android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java]: done

File: android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java [hunk 2]
```diff
@@ -26,8 +26,9 @@ protected OTSubscriberLayout createViewInstance(ThemedReactContext reactContext)
 
     @ReactProp(name = "streamId")
     public void setStreamId(OTSubscriberLayout view, String streamId) {
-
+        if(streamId != null) {
```
Comment (ggoldens) [inline android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java:29]: I think the indentation of this if block looks wrong.

**Review by ggoldens — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java (hunk 2 above)
Comment (0x1bitcrack3r) [reply android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java:29]: the indentation is correct

**Review by ggoldens — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java (hunk 2 above)
Comment (0x1bitcrack3r) [reply android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java]: Hi I have formatted as you said please check

**Review by ggoldens — APPROVED** [review]
LGTM

**Pull request merged**